src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/dec/BrotliStream.Decompress.cs

```diff
@@ -27,6 +39,14 @@ public override int ReadByte()
             return numRead != 0 ? b : -1;
```

Comment:
The method `ReadByte` did not have a documentation item in the xml in Docs, for the reasons described in this issue I opened:
https://github.com/dotnet/runtime/issues/46712